scripts/lib/upgrade-zulip-stage-2

```diff
@@ -69,67 +70,93 @@ if (vendor, os_version) in UNSUPPORTED_DISTROS:
 os.umask(0o22)
 
 parser = argparse.ArgumentParser()
-parser.add_argument("deploy_path", metavar="deploy_path",
-                    help="Path to deployment directory")
-parser.add_argument("--skip-puppet", dest="skip_puppet", action='store_true',
-                    help="Skip doing puppet/apt upgrades.")
-parser.add_argument("--skip-migrations", dest="skip_migrations", action='store_true',
-                    help="Skip doing migrations.")
-parser.add_argument("--from-git", dest="from_git", action='store_true',
-                    help="Upgrading from git, so run update-prod-static.")
-parser.add_argument("--ignore-static-assets", dest="ignore_static_assets", action='store_true',
-                    help="Do not attempt to copy/manage static assets.")
-parser.add_argument("--skip-purge-old-deployments", dest="skip_purge_old_deployments",
-                    action="store_true", help="Skip purging old deployments.")
+parser.add_argument("deploy_path", metavar="deploy_path", help="Path to deployment directory")
+parser.add_argument(
+    "--skip-puppet",
+    dest="skip_puppet",
+    action="store_true",
+    help="Skip doing puppet/apt upgrades.",
+)
+parser.add_argument(
+    "--skip-migrations", dest="skip_migrations", action="store_true", help="Skip doing migrations.",
+)
+parser.add_argument(
+    "--from-git",
+    dest="from_git",
+    action="store_true",
+    help="Upgrading from git, so run update-prod-static.",
+)
+parser.add_argument(
+    "--ignore-static-assets",
+    dest="ignore_static_assets",
+    action="store_true",
+    help="Do not attempt to copy/manage static assets.",
+)
+parser.add_argument(
+    "--skip-purge-old-deployments",
+    dest="skip_purge_old_deployments",
```

Comment:
These `dest` arguments are useless as they are the default, right?  Maybe we should fix that across our `argparse` usage as I think it may be responsible for most of the cases where `add_argument` becomes giant.